src/preattestation.rs

```diff
@@ -350,9 +356,9 @@ mod idblock {
 
         // Formatted in Base-64 since it's the format QEMU takes.
         let id_block_string =
-            general_purpose::STANDARD.encode(bincode::serialize(&measurements.id_block)?);
+            general_purpose::STANDARD.encode(measurements.id_block.to_bytes()?.as_ref());
```

Comment:
Can just use the reference symbol instead of as_ref()
```suggestion
            general_purpose::STANDARD.encode(&measurements.id_block.to_bytes()?);
```